### PR TITLE
Add Kitematic documentation to docs.master.docker.com

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit SvenDowideit@home.org.au

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@bfirsh @jeffdm @docker/docs-owners 

once this merges I'll make a docs branch update PR to match.

File: docs/Dockerfile [hunk 1]
```diff
@@ -116,5 +117,47 @@ ADD https://raw.githubusercontent.com/docker/compose/${COMPOSE_BRANCH}/docs/inde
...
+#######################
+ADD https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/faq.md \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/index.md \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/known-issues.md \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/minecraft-server.md \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/nginx-web-server.md \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/rethinkdb-dev-database.md \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/userguide.md \
+  /docs/sources/kitematic/
+RUN sed -i.old '1s;^;no_version_dropdown: true;' /docs/sources/compose/*.md
```
Comment (thaJeztah) [inline docs/Dockerfile]: Copy pasta? Looks like this line is duplicated from line 118

File: docs/sources/installation/mac.md [hunk 2]
```diff
@@ -8,14 +8,13 @@ You can install Docker using Boot2Docker to run `docker` commands at your comman
 Choose this installation if you are familiar with the command-line or plan to
 contribute to the Docker project on GitHub.
 
+[<img src="/installation/images/kitematic.png" alt="Download Kitematic"
```
Comment (thaJeztah) [inline docs/sources/installation/mac.md]: Rather "interesting" way to combine HTML with Markdown; particular reason for not using a plain `<a href=...` here?

No show-stopper, just a bit odd construct

File: docs/Dockerfile (hunk 1 above)
Comment (thaJeztah) [inline docs/Dockerfile]: Should probably be `/docs/sources/kitematic/`

File: docs/Dockerfile [hunk 3]
```diff
@@ -116,5 +117,47 @@ ADD https://raw.githubusercontent.com/docker/compose/${COMPOSE_BRANCH}/docs/inde
...
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/assets/nginx-data-folder.png \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/assets/nginx-data-volume.png \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/assets/nginx-hello-world.png \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/assets/nginx-preview.png \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/assets/nginx-serving-2048.png \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/assets/rethink-container.png \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/assets/rethink-create.png \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/assets/rethink-ports.png \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/assets/rethinkdb-preview.png \
+  https://raw.githubusercontent.com/kitematic/kitematic/${KITEMATIC_BRANCH}/docs/assets/volumes-dir.png \
```
Comment (thaJeztah) [inline docs/Dockerfile]: Just a note: I think at some point it may be cleaner to define an `ENV` for the repos as well

**Comment by moxiegirl** [comment]
@thaJeztah You make a good point. This will all change with the release retooling though and much of it won't be exposed to contributors; it will be in our build scripts.

**Comment by moxiegirl** [comment]
LGTM

**Comment by thaJeztah** [comment]
@moxiegirl my first comment should probably be checked (although I don't know if its nescessary for the kitematic docs, and probably won't do anything harmful when ran twice); https://github.com/docker/docker/pull/13315#discussion_r30571447

Otherwise, LGTM, so feel free to merge if it's no problem

**Comment by fredlf** [comment]
LGTM once copy pasta is fixed. 

Now I'm hungry.

File: docs/sources/installation/mac.md (hunk 2 above)
Comment (SvenDowideit) [inline docs/sources/installation/mac.md]: there is a reason :) mkdocs will correctly link up the markdown to the `/kitematic/` at the `.md` file level - whereas using the html link, mkdocs ignores is as a raw html link.

File: docs/Dockerfile (hunk 3 above)
Comment (SvenDowideit) [inline docs/Dockerfile]: i was pondering that too

**Comment by SvenDowideit** [comment]
roger that, I'll make fixes very soon and merge and get it the Kitematic guys "published' :)

File: docs/Dockerfile (hunk 1 above)
Comment (SvenDowideit) [inline docs/Dockerfile]: yup, much pasta - home made of course.

File: docs/sources/installation/mac.md (hunk 2 above)
Comment (thaJeztah) [inline docs/sources/installation/mac.md]: Ah!

**Pull request merged**